virttest/qemu_vm.py

```diff
@@ -1053,8 +1053,14 @@ def add_memorys(devices, params):
                 name = output[0]
                 backend_options = {}
                 backend_options["size_mem"] = "%sM" % params["mem"]
-                if params.get("vm_mem_backend"):
-                    backend_options["backend_mem"] = params.get("vm_mem_backend")
+                mem_backend = params.get("vm_mem_backend")
+                if mem_backend:
+                    backend_options["backend_mem"] = params["vm_mem_backend"]
+                if (not params.get("vm_mem_mem-path") and mem_backend ==
+                        "memory-backend-file"):
+                    LOG.error("expect mem-path for %s", mem_backend)
```

Comment:
As this is additional info, we keep the wrong usage for negative test, just logging it.